style.css

```diff
@@ -0,0 +1,268 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700&display=swap");
+
+:root {
+  --primary-color: #36b37e;
+  --secondary-color: #172b4d;
+  --primary-font: "Poppins", sans-serif;
+  --secondary-font: "Roboto", sans-serif;
+  --tertiary-font: "Inter", sans-serif;
+  --skills-list-bg: #ebf0ee;
+  --skills-list-color: #3a4a42;
+  --p-text-color: #42526e;
+}
+
+.main-container {
+  display: flex;
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  top: 1px;
+  align-content: space-between;
+  background: #fff;
+}
+
+/* shapes --- ellips and regtangles */
+.bg-shapes {
+  background-image: url(img/shapes.png);
+  position: absolute;
+  width: 462px;
+  height: 587.15px;
+  top: 104px;
+  margin: 0 auto;
+}
+
+/* top bar section */
+.top-bar {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+}
+
+/* my logo and three dashs */
+.tool-bar {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 44px;
+  bottom: 0;
+  height: 48px;
+  width: 375px;
+  border-radius: 0;
+}
+
+.dash-setting {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  right: 24px;
+  top: calc(50% - 40px / 2);
+  border-radius: 8px;
+}
+
+.icons {
+  width: 24px;
+  height: 24px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.icon-Union {
+  position: absolute;
+  width: 18px;
+  height: 12px;
+  left: 3px;
+  top: 6px;
+
+  /* background: #6070FF; */
+}
+
+/* where the Logo Placed */
+
+.logo-setting {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: absolute;
+  width: 100px;
+  height: 40px;
+  left: 12px;
+  top: calc(50% - 40px / 2);
+  border-radius: 0;
+  padding: 10px 12px 10px 12px;
+}
+
+/* My Logo */
+
+.logo {
+  width: 76px;
+  height: 20px;
+  font-family: var(--primary-font);
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+/* /* Headline */
+
+.content-section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px 114px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 604px;
+  left: calc(50% - 375px / 2);
+  top: 25px;
+}
```

Comment:
[Optional] It would be nice if your headline section matched the figma template better🙏. Currently you could add some spacing to your text to make it look better. Check out the comparison below:

Your design:
<img width="299" alt="image" src="https://user-images.githubusercontent.com/37116322/221351579-b2158318-2f69-4138-8cd8-b062bbb8e494.png">


Figma design:
<img width="197" alt="image" src="https://user-images.githubusercontent.com/37116322/221351622-2b59406c-1d83-40a3-956e-033486a5293d.png">

A few things to note:
- You can reduce the font-size of your main title. I think `40px` would be better
- You can add a little more padding between the paragraph and `lets connect` and between `lets connect` and the social icons
- Your social media icons should be aligned to the left. Same as the text
- Your twitter icon is currently larger than the rest. Please make all of them the same size
- You can stop your background image from repeating by adding the `background-repeat: no-repeat` style to the image